src/Components/WebAssembly/Samples/HostedBlazorWebassemblyApp/Client/HostedBlazorWebassemblyApp.Client.csproj

```diff
@@ -15,4 +15,17 @@
     <ProjectReference Include="..\Shared\HostedBlazorWebassemblyApp.Shared.csproj" />
   </ItemGroup>
 
+  <Target Name="_SkipXmlDocFiles" BeforeTargets="_ResolveBlazorWasmOutputs">
+    <ItemGroup>
+      <ReferenceCopyLocalPaths Remove="@(ReferenceCopyLocalPaths)" Condition="'%(ReferenceCopyLocalPaths.Extension)' == '.xml'"></ReferenceCopyLocalPaths>
+    </ItemGroup>
+  </Target>
+
+  <Target Name="_FixUpDotnetJsFile" BeforeTargets="_ResolveBlazorWasmOutputs" AfterTargets="_ScrambleDotnetJsFileName">
```

Comment:
Will the various `_FixUpDotnetJsFile` targets also be removed when `_SkipXmlDocFiles` is removed?

Reply:
Yep

Reply:
Is there an issue to undo this?

Reply:
I picked up an updated SDK so this is undone